src/status_im/contexts/onboarding/share_usage/view.cljs

```diff
@@ -0,0 +1,60 @@
+(ns status-im.contexts.onboarding.share-usage.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [react-native.safe-area :as safe-area]
+    [status-im.common.events-helper :as events-helper]
+    [status-im.common.resources :as resources]
+    [status-im.contexts.onboarding.share-usage.learn-more-sheet :as learn-more-sheet]
+    [status-im.contexts.onboarding.share-usage.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- share-usage-data-fn
+  [enabled? next-screen]
+  (rf/dispatch [:centralized-metrics/toggle-centralized-metrics enabled? true])
+  (rf/dispatch [:navigate-to next-screen]))
```

Comment:
Used `navigate-to` instead of `navigate-to-within-stack` - https://github.com/status-im/status-mobile/pull/21655/files#r1856286497